src/main/java/de/eldoria/schematicbrush/commands/parser/BrushSettingsParser.java

```diff
@@ -234,15 +241,42 @@ private BrushSettingsParser() {
         Matcher matcher = ArrayUtil.findInArray(args, Y_OFFSET);
         if (matcher != null) {
             String value = matcher.group(1);
-            int offset;
-            try {
-                offset = Integer.parseInt(value);
-            } catch (NumberFormatException e) {
-                MessageSender.getPluginMessageSender(SchematicBrushReborn.class)
-                        .sendError(player, "Invalid offset.");
-                return Optional.empty();
+
+            if (value.startsWith("[") && value.endsWith("]")) {
+                String stripped = value.substring(1, value.length() - 1);
+                if (stripped.contains(":")) {
+                    String[] split = stripped.split(":");
+                    OptionalInt min = Parser.parseInt(split[0]);
+                    OptionalInt max = Parser.parseInt(split[1]);
+                    if (!(min.isPresent() && max.isPresent()) || min.getAsInt() > max.getAsInt()) {
+                        MessageSender.getPluginMessageSender(SchematicBrushReborn.class)
+                                .sendError(player, "Invalid offset.");
+                        return Optional.empty();
+                    }
+                    settingsBuilder.withYOffset(IOffset.range(min.getAsInt(), max.getAsInt()));
+                } else if (stripped.contains(",")) {
+                    String[] stringNumbers = stripped.split(",");
```

Comment:
`1,2,,,` would be a valid input string - not sure if this is wanted

Reply:
It would be, but cant be parsed because the numbers would be invalid. I think this is fine.

Reply:
It can be parsed, the returned array for my example would be `["1", "2"]`

Reply:
Oh well... I thought split produces some empty strings there, but it doesnt.

But what would you expect to happen? Throwing away any `,` without a following number or throwing an error, because a number is missing?

Reply:
Anyway. 
I just noticed that a `,,` would be technically possible, but wouldn't be recognized by the regex as a valid offset.
However I adjusted the regex to also ignore any trailing `,` since this would be invalid syntax as well.

https://regex101.com/r/uwML0K/1

Reply:
I made the following changes:
Adjust regex to match only valid flags
Add error when a invalid flag value is given. The old parser would just ignore them.
Removed a exception on an invalid flag value, which wasn't even catched (wth?)
Enhanced the tabcompletion a bit.

Reply:
[String#split(String, int)](https://docs.oracle.com/en/java/javase/16/docs/api/java.base/java/lang/String.html#split(java.lang.String,int)) allows to force parse empty strings in such cases

Reply:
> [String#split(String, int)](https://docs.oracle.com/en/java/javase/16/docs/api/java.base/java/lang/String.html#split(java.lang.String,int)) allows to force parse empty strings in such cases

But in this case I would need to count the occurrences of `,` first in order to set the limit. The regex is fine I think and also fixes this issue ^^

Reply:
> If the *limit* is negative then the pattern will be applied as many times as possible and the array can have any length.

This produces an array with empty strings in the given case

Reply:
Anyway qwq it works now qwq